docs/literate/main.jl

```diff
@@ -6,7 +6,9 @@ using CairoMakie
 using Statistics
 using LuxCUDA
 using Revise
+using testdata
 includet("../testdata.jl")
+# Create an object of the MersenneTwister, use this object to create random number between 0 and 1
```

Comment:
to clarify: MersenneTwister is an algorithm to generate random numbers, but the actual random number function to create numbers between 0 and 1 is simply `rand`